.github/workflows/gh-pages.yml

```diff
@@ -129,12 +129,13 @@ jobs:
 
       - name: Build | Rust Docs
         run: |
+          mkdir -p build/rust
           cd rust/hacl-sys
           cargo doc --no-deps
-          cp -r target/doc build/rust/main
+          cp -r target/doc ../../build/rust/main
           cd ../
-          cargo doc --no-deps
-          cp -r target/doc build/rust/main
+          cargo doc --no-deps --all-features
+          cp -r target/doc/* ../build/rust/main
```

Comment:
This looks a bit off on first glance. But let's see if it does what it should.

Reply:
Right, all-features enables the non standard features like hazmat.
There is no real merging of folders. The top level only holds js stuff, the actual crate docs are in sub folders (hacl and hacl-sys)

Reply:
Thanks for clarifying.